packages/micropip/src/micropip/micropip.py

```diff
@@ -21,10 +22,11 @@
     IN_BROWSER = False
 
 if IN_BROWSER:
-    # In practice, this is the `site-packages` directory.
-    WHEEL_BASE = Path(__file__).parent
+    import site
+
+    WHEEL_BASE = site.getsitepackages()[0]
 else:
-    WHEEL_BASE = Path(".") / "wheels"
+    WHEEL_BASE = str(Path(".") / "wheels")
```

Comment:
Probably better to keep paths as `Path` (i.e. make the one above a Path as well). It's used in `Zipfile.extractall` which should support it.